### PR TITLE
Feat/loading screen

### PR DESCRIPTION
**Pull request by attom2**
# Pull request checklist

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Current behavior

- No loading screen when user fills out meta mask info

## New behavior

- Loading screen, new logo

## Other information

image
image

### CODE REVIEW

File: src/components/Loading.vue [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+}
+
+@keyframes lds-ring {
+  0% {
+    transform: rotate(0deg);
+  }
+  100% {
+    transform: rotate(360deg);
+  }
+}
```
Comment (edmdc) [inline src/components/Loading.vue:145]: How did you come about this?

**Review by jpc20 — APPROVED** [review]
Looks great Andy! I have been looking into estimating the transaction time, but I haven't had any luck. It seems like something well adjust after everyone tries it on their systems.

File: src/components/Loading.vue [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+    let j = 0
+    setInterval(
+      /* eslint-disable-next-line */
+      function() { 
+        self.mainImageSrc = self.images[i].image;
+        i += 1
+        if (i === 4) {
+          i = 0
+        }
+      }, 1000
```
Comment (jpc20) [inline src/components/Loading.vue:86]: This is beautiful, I wish I could write JS like this.

**Review by edmdc — APPROVED** [review]

File: src/components/Loading.vue (hunk 2 above)
Comment (attom2) [reply src/components/Loading.vue:86]: Lol me too, stack overflow helps a lot

**Pull request merged**